string team
string robot
duration skew
time beacon